01-Setup and Getting started/Readme.md

```diff
@@ -106,6 +106,16 @@ The CSV file and LOG file are generated under a org-specific sub-folder in the f
 E.g.  
 C:\Users\<UserName>\AppData\Local\Microsoft\AzSK.ADOLogs\Org_[yourOrganizationName]\20181218_103136_GADS
 
+## Customize location for scan reports
+The path for saving scan reports can be customized using below command:
```

Comment:
The location to save scan reports can now be customized using the command below:

Reply:
updated the statement